src/util/segmentation/operations/getPixelPathBetweenPixels.js

```diff
@@ -0,0 +1,261 @@
+import external from '../../../externalModules.js';
+
+const oneOverRoot2 = 1 / Math.sqrt(2); // Cache this to avoid repeated computation.
```

Comment:
I can't find where this file/method/operation is ever used?

Reply:
In `src/tools/segmentation/strategies/correction.js`.